build.gradle

```diff
@@ -66,8 +66,8 @@ dependencies {
             "ch.qos.logback:logback-classic:$logbackVersion",
             'org.mockito:mockito-core:1.10.19',
             "org.assertj:assertj-core:3.8.0",
-            "org.web3j:core:$web3jVersion:tests"
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
+            "org.web3j:core:$web3jVersion:tests",
+            "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```

Comment:
This has now been moved to a `testCompile` dependency, it should be under `compile`